dd-sdk-android/src/main/kotlin/com/datadog/android/v2/core/internal/storage/Storage.kt

```diff
@@ -24,11 +24,17 @@ internal interface Storage {
     //   request transformation).
     /**
      * Utility to write data, asynchronously.
+     * @param forceNewBatch if `true` will force the writer to start a new batch before storing the
+     * data. By default this flag is `false`.
      * @param callback an operation to perform with a [EventBatchWriter] that will target the current
      * writeable Batch
      */
     @AnyThread
-    fun writeCurrentBatch(datadogContext: DatadogContext, callback: (EventBatchWriter) -> Unit)
+    fun writeCurrentBatch(
+        datadogContext: DatadogContext,
+        forceNewBatch: Boolean = false,
```

Comment:
do we need to have the default value here if in all the calls (apart of the tests) we will get the value explicitly?